src/pip/_internal/locations/_distutils.py

```diff
@@ -87,6 +83,7 @@ def distutils_scheme(
             prefix = i.install_userbase  # type: ignore
         else:
             prefix = i.prefix
+        assert prefix is not None
```

Comment:
Probably better to annotate `prefix` as `str` instead.

Reply:
Right now the argument is defined as `prefix: str = None`. Are you suggesting to make the default an empty string or to make the argument required?